config/locales/en.yml

```diff
@@ -829,72 +829,72 @@ en:
           "yes": "Bridge"
         building:
           apartment: "Apartment"
-          apartments: "Apartments"
+          apartments: "Apartment Building"
           barn: "Barn"
           bungalow: "Bungalow"
           cabin: "Cabin"
-          chapel: "Chapel"
+          chapel: "Chapel Building"
           church: "Church Building"
           civic: "Civic Building"
           college: "College Building"
           commercial: "Commercial Building"
-          construction: "Building under Construction"
+          construction: "Building Under Construction"
           cowshed: "Cowshed"
           detached: "Detached House"
           dormitory: "Dormitory"
           duplex: "Duplex House"
-          farm: "Farm House"
-          farm_auxiliary: "Auxiliary Farm House"
+          farm: "Farmhouse"
+          farm_auxiliary: "Farm Building"
           garage: "Garage"
           garages: "Garages"
           greenhouse: "Greenhouse"
-          hangar: "Hangar"
+          hangar: "Hangar Building"
```

Comment:
I usually do not hear fellow American English speakers say "building" after "hangar."

Reply:
Yeah, this came from id-tagging-schema, which needs to distinguish between the presets for `aeroway=hangar` and `building=hangar`. But that distinction is probably less important in the context of these search results.